content/en/docs/ops/deployment/requirements/index.md

```diff
@@ -60,22 +60,27 @@ requirements:
 
 ## Ports used by Istio
 
-The following ports and protocols are used by Istio.
+The following ports and protocols are used by Istio sidecar proxy.
 
-| Port | Protocol | Used by | Description |
+| Port | Protocol | Description | Local host only |
```

Comment:
What does "local host only" (true or false) mean? Why should readers care?

Reply:
I think this is useful. It tells user which port can be visited from the outside. Although I see your point that local host only is kind of confusing here.. @costinm @howardjohn any suggestion on the naming?

Reply:
From outside of what? Outside of the pod? If so, how about:
```suggestion
| Port | Protocol | Description | Pod-internal only |
```

Reply:
"local host only", "pod internal", or "Exposed externally" (and inverting) all sound equally good to me

Reply:
In that case, I recommend going with "Pod-internal". It's the least ambiguous IMO.